pattern-1/apim/wso2apim-deployment.yaml

```diff
@@ -81,9 +81,9 @@ spec:
         - name: apim-storage-volume
           mountPath: /home/wso2carbon/wso2am-2.5.0/repository/deployment/server
```

Comment:
@ThilinaManamgoda the product pack version of the mount needs to change to 2.6.0.